lib/Widget/Provider/DataProvider.php

```diff
@@ -153,6 +166,22 @@ public function getDisplayLongitude(): float
         return $this->longitude;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function isPreview(): bool
+    {
+        return $this->isPreview;
```

Comment:
Could this be as simple as `return !empty($this->displayId)` so that we didn't have to have `setPreview`, etc ?